tests/openassetio-python/CMakeLists.txt

```diff
@@ -195,35 +195,41 @@ openassetio_add_pytest_target(
 #-----------------------------------------------------------------------
 # CTest setup fixtures
 
-# Fixture: create Python environment.
-openassetio_add_test_fixture_target(openassetio-python-venv)
+# Optional fixture dependency to create a Python environment in the
+# install tree.
+set(_venv_if_enabled )
+if (OPENASSETIO_ENABLE_PYTHON_TEST_VENV)
+    # Fixture: create Python environment.
+    openassetio_add_test_fixture_target(openassetio-python-venv)
+    set(_venv_if_enabled openassetio-python-venv)
+endif ()
+
 
 # Fixture: install codegen.
 openassetio_add_test_fixture_target(openassetio.internal.resources-codegen.install)
 openassetio_add_test_fixture_dependencies(
     openassetio.internal.resources-codegen.install
-    openassetio-python-venv
+    ${_venv_if_enabled}
 )
 
 
 #-----------------------------------------------------------------------
 # CTest test targets
 
-
 # Core Python component tests.
 openassetio_add_test_target(openassetio.internal.python-pytest)
 openassetio_add_test_fixture_dependencies(
     openassetio.internal.python-pytest
     openassetio.internal.install
-    openassetio-python-venv
+    ${_venv_if_enabled}
 )
 
 # Examples tests.
 openassetio_add_test_target(openassetio.internal.resources-examples-pytest)
 openassetio_add_test_fixture_dependencies(
     openassetio.internal.resources-examples-pytest
     openassetio.internal.install
-    openassetio-python-venv
+    ${_venv_if_enabled}
```

Comment:
[question] Do we need this for codegen too at this point?

Reply:
It's a transitive dependency of installing the codegen deps.  Should we make it explicit there too? Might help readability and does no harm

Reply:
Indeed it was needed for the codegen tests too, since the codegen deps are installed as part of the `openassetio-python-venv` target.  This was added as part of a squashed fixup to d95c55ba.